docs/artifact-visualization.md

```diff
@@ -10,6 +10,7 @@ Use cases:
 * Visualizing end results of ML pipeline runs.
 * Debugging workflows where visual artifacts are the most helpful.
 
+> **Click image to play demo video**      
 [![Demo](https://img.youtube.com/vi/whoRfYY9Fhk/0.jpg)](https://youtu.be/whoRfYY9Fhk)
```

Comment:
```suggestion
> Click the image below to play a demo video

[![Demo](https://img.youtube.com/vi/whoRfYY9Fhk/0.jpg)](https://youtu.be/whoRfYY9Fhk)
```

a new line is needed here as in the current screenshot, the image is _within_ the blockquote.
bold on top of the blockquote is not necessary, IMO.
also removed the trailing spaces that are causing lint to fail

Reply:
Thank you for the helpful advice!!!